src/components/Modal.svelte

```diff
@@ -0,0 +1,21 @@
+<script lang="ts">
+  export let close = () => {};
+</script>
+
+<div
+  on:click={close}
+  on:keyup={close}
+  role="button"
+  tabindex="0"
```

Comment:
All of this stuff is for accessibility standards. If curious, you can look it up in the svelte docs.